src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -57,6 +61,60 @@ def _get_start_and_end_indices(data: list[TimeSeriesPoint]) -> tuple[int, int]:
     return start, end
 
 
+def handle_send_historical_data_to_seer(alert_rule, project):
```

Comment:
Feel free to propose a new name for this 😅 I was having a hard time

Reply:
`ensure_successful_seer_request`? 🤔 i would name it related to the exception / error handling around the send.

honestly, i think what you have is probably fine, `handle` is just overloaded for me with FE stuff 🤣